src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -161,9 +161,12 @@ Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Body.get -> Micro
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Keyword.get -> Microsoft.CodeAnalysis.SyntaxToken
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.Modifiers.get -> Microsoft.CodeAnalysis.SyntaxTokenList
 Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.SemicolonToken.get -> Microsoft.CodeAnalysis.SyntaxToken
+Microsoft.CodeAnalysis.CSharp.Syntax.AccessorDeclarationSyntax.ExpressionBody.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ArrowExpressionClauseSyntax
```

Comment:
I will correct that in one of the next pushes. I have to run tests with Jenkins first, because my exceptions are in German and that causes confusion.